moveit_ros/benchmarks/package.xml

```diff
@@ -17,13 +17,20 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>libboost-dev</build_depend>
+  <build_depend>libboost-date-time-dev</build_depend>
```

Comment:
Instead of referencing these packages explicitly, I think we should just use `<depend>boost</depend>` (for libboost-all-dev) just as with `moveit_core`.

Reply:
The point I believe of these changes is actually do *reduce* the size of dependencies. The `boost` key is almost a wildcard when it comes to Boost dependencies, dragging up to 700MB+ of additional (and often unnecessary) dependencies.

See [Generating ‘dev’ and runtime artefacts from ROS packages](https://discourse.ros.org/t/generating-dev-and-runtime-artefacts-from-ros-packages/12448) for some context.

Reply:
I see, in general it's definitely much better to include all dependencies explicitly, but then I think we should fix this for all packages (including upstream, i.e. ompl) and for MoveIt 1/2 the same way.

Reply:
> See [Generating ‘dev’ and runtime artefacts from ROS packages](https://discourse.ros.org/t/generating-dev-and-runtime-artefacts-from-ros-packages/12448) for some context.

Thanks, very interesting findings!

Any reasons to not consolidate build/exec depends in a single `<depend>` tag?

Reply:
> Any reasons to not consolidate build/exec depends in a single `<depend>` tag?

I don't understand your question.

Aren't the reasons to use fine-grained permissions discussed in the thread I (and you) link to?

Reply:
Well, the major improvement seems to be coming from not using boost-all, dev/non-dev separation seems to only really come into play when creating dev/non-dev ROS packages (which would be a great thing to do eventually). I'm all in favor of specifying dependencies as accurate (or minimal) as possible, but then I would prefer a more coordinated refactoring for all repos/branches.

Reply:
Taking small steps is probably more feasible though.

Reply:
I'm all for being explicit with dependencies and smaller install footprints.